### ISSUE TITLE
Mycetoma monthly meeting 24 January 2023

### ISSUE THREAD

**Issue by wwjvdsande** [OpenSourceMycetoma/Series-1-Fenarimols#85]
Time: January 24th (12 noon UK, 1pm EU, 11pm Aus) [Timezones](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2022&month=11&day=22&hour=12&min=0&sec=0&p1=43&p2=136&p3=37&p4=771&p5=235&p6=152&p7=770&p8=250&p9=196&p10=240).

The Outlook invite can be found [here](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/9674081/Monthly.MycetOS.meeting00.zip)

Location: https://uni-sydney.zoom.us/j/99281549497. In case of failure, use https://ucl.zoom.us/j/99789829423 and in case of double failure use [MycetOS meeting teams link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2I2MDcxZTgtYjZjOS00YTdmLTgwZTMtNWQ3OTc4NDY0MDNj%40thread.v2/0?context=%7b%22Tid%22%3a%22526638ba-6af3-4b0f-a532-a1a511f4ac80%22%2c%22Oid%22%3a%22a5ed84b8-5af1-42c6-9f8d-240f23a5d997%22%7d)
Chair: USyd
Minute taker: ErasmusMC
Previous meeting: https://youtu.be/gRVsCUZWUWEv
Recording: [here]

AGENDA
1. Outstanding items from previous meeting
- Paper writing
- Statistics vivo data
2. Rohan's contribution of compounds from Griffith University QLD: Numbers of compounds, screening capacity, costs and motifs of interest, to be discussed.
Crowdsourcing update - incoming contributions from e.g. Whitgift school, Hampton school, Julia Sarju (@JuliaSarju, York Uni). Others @kym834 ?
3. Updates:
USYD update
UCL update
Erasmus update

4. Any other businesses?

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib I made a new agenda for the next meeting. Based on the minutes from previous meeting I simply rotated the chair and the minute taker. If it needs to be different, please correct. Hope to see you all next Tuesday!

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib: @MA-Jjingyi and @wwjvdsande have updated the spreadsheet with the in vitro susceptibility data.

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi @MFernflower Two new niclosamide derivatives 4SF5-Closamide and 4SCF3-Closamide are quite promising. Might be candidates for toxicity testing in the next summer.
Maybe you can test the missing fragrances (maritima, nootkatone) and metal compounds until spring. I could write a paper about the fragrances then.
@wwjvdsande mentioned that methylene blue was investigated as part of a photodynamic therapy before. Is there a publication about that?

**Comment by wwjvdsande** [comment]
@bebi78 no we never published the methylene blue study. It was quite tricky with the light we used. it did seem to work though but only under the special light

**Comment by bebi78** [comment]
@wwjvdsande Yes, the absorption maximum of MB is 664 nm, so lasers emitting in that range should be applied to get maximum phototoxic activity. Because MB in the ´´dark´´ or daylight already has an IC50 value of 2.1 µM, I think that submicromolar activities can be reached upon irradiation with a proper laser.

**Comment by MFernflower** [comment]
@bebi78 niclosamide seems promising - I guess it would be up to DNDI to work on clinical trials for it for mycetoma 

seeing the 4SF5-Closamide gave me this idea for a molecule though: 
 4-Ethyl-Closamide
![image](https://user-images.githubusercontent.com/3164942/213896316-71d96090-2209-45ce-b712-e691260c20d9.png)

**Comment by bebi78** [comment]
@wwjvdsande @MFernflower Dr. Wainwright from the University of Liverpool has developed a promising lipophilic MB derivative (S137, see link below to a recent paper using G. mellonella fungal infection model). I think we should ask him for a sample of S137.
https://link.springer.com/article/10.1007/s43630-022-00258-4

**Comment by MFernflower** [comment]
"Our university compound database says that the 4-ethyl and 4-butylaniline precursors are available at our local Macromolecules department. So I will try to get them and prepare these compounds."

Sounds good!

**Comment by fantasy121** [comment]
[Addn'l Epichem Compounds Refined List.zip](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/files/10486334/Addn.l.Epichem.Compounds.Refined.List.zip)
![Addn'l Epichem Compounds Refined List](https://user-images.githubusercontent.com/36176821/214212616-6c85789d-1a68-48dc-8117-21e972c39dd3.jpeg)

**Comment by fantasy121** [comment]
I refined the list of gap compounds to 13, please see above.

Re: draft, my writing is completed.

**Comment by fantasy121** [comment]
Sorry I won't make it to today's meeting. Please send my apologies. I will catch up afterwards, just tag me if there's any action item for me thanks.

**Comment by wwjvdsande** [comment]
@fantasy121 will someone from USyd join? If not that we should use the link from UCL.

**Comment by mattodd** [comment]
Hi @fantasy121 for the additional Epichem compounds you've selected above, can you please just walk us through
1) Why those, i.e. what SAR questions will they help us answer, and
2) Whether there are any important SAR gaps that you think remain unanswered, and whether we need to make any compounds to answer them.
This helps us justify any extra request to Epichem (which comes with a resources cost) as well as plan any remaining synthesis that we think is unavoidable.

**Comment by mattodd** [comment]
> @fantasy121 will someone from USyd join? If not that we should use the link from UCL.

...and we'd need an alternative chair unless @kym834 or @alintheopen can join. If not, I can do this, given that Erasmus is minute-taker.

**Comment by mattodd** [comment]
OK, let's use UCL link https://ucl.zoom.us/j/99789829423
